src/components/Footer.js

```diff
@@ -161,23 +161,34 @@ const Footer = () => {
       </StyledFooter>
       <div css={tw`bg-gold`}>
         <div className="container" css={tw`py-2 text-center font-bold`}>
-          If you are feeling suicidal or in emotional distress, call the{' '}
-          <OutboundLink
-            to="https://suicidepreventionlifeline.org"
-            eventLabel="Suicide prevention link from footer"
-            css={tw`underline`}
-          >
-            Suicide Prevention Lifeline
-          </OutboundLink>{' '}
-          at{' '}
-          <OutboundLink
-            eventLabel="Suicide prevention hotline link from footer"
-            to="tel:1-800-273-8255"
-            css={tw`underline whitespace-no-wrap`}
-          >
-            1-800-273-8255
-          </OutboundLink>{' '}
-          | In emergencies, dial 911.
+          <p css={tw`sm:hidden`}>
```

Comment:
The mobile menu begins to appear at the `md` breakpoint - so in order to keep this in sync with that, we should use:
```suggestion
          <p css={tw`md:hidden`}>
```
I think visually it might make be good to add some space between the helpline and suicide prevention paragraphs so we could also add a `mb-4` (margin bottom of 4 units).